src/library_browser.js

```diff
@@ -948,7 +948,7 @@ var LibraryBrowser = {
     return 0;
```

Comment:
Yup, I guess we would need specific coverage each function in isolation to catch this kind of thing.

Perhaps we could find a way to allow the `makeDynCall` helper to add deps to the current/calling function.. although that could be tricky.

Reply:
I decided to create a method to automatically inject these dependencies when needed.   It could possible be extended to all library functions, but for now I've limited it to just these two.  Makes the change way smaller and easier to audit.